packages/postgress/src/index.ts

```diff
@@ -0,0 +1 @@
+export { PostgresMessagePickupRepository } from './PostgresMessagePickupRepository'
```

Comment:
We need to export PostgresMessagePickupRepositoryConfig and ConnectionInfo interface as well

Reply:
Solved!